test/ASTMerge/class-template/Inputs/class-template3.cpp

```diff
@@ -0,0 +1,12 @@
+template<typename _T>
+struct Struct {
+};
+inline void Func(int x,Struct<int> &y){
```

Comment:
Do we need the inline keyword for the issue to be triggered? If no, I prefer to remove every redundant bit.

Reply:
Also, the names of the functions should start with lowercase letters.

Reply:
inline is not needed

how to format, in this OK?
```
template <typename T>
struct Y {
};
void f1(int, Y<int> &y) {
}
template <>
struct Y<int> {
  void f2() {
    Y<int> a;
    f1(0, a);
  }
};
```